web/modules/custom/joinup_community_content/src/CommunityContentWorkflowAccessControlHandler.php

```diff
@@ -411,9 +373,16 @@ protected function getPermissionScheme(string $operation): array {
    *
    * @return \Drupal\node\NodeStorageInterface
    *   The storage handler.
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   *   Thrown if the entity type doesn't exist.
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
+   *   Thrown when an entity with a non-existing storage is passed.
```

Comment:
Just for the record, I kinda hate these exceptions in this place.. If the storage is wrong or the entity type doesn't exist, then something is very very broken in the website and it will not wait until the access control handler to throw the exception.. Unless it is a typo, which should not be a typo... Anyway, since it is defined, we should place them.

Reply:
It just annoys me to see them in the IDE